src/serac/physics/solid_mechanics.hpp

```diff
@@ -459,6 +459,27 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
     bcs_.addEssential(disp_bdr, disp_bdr_coef_, displacement_.space());
   }
 
+  /**
+   * @brief Set the displacement essential boundary conditions on a single component
+   *
+   * @param[in] disp_bdr The set of boundary attributes to set the displacement on
+   * @param[in] disp The vector function containing the set displacement values
+   * @param[in] component The component to set the displacment on
+   *
+   * For the displacement function, the argument is the input position, the second argument is the time, and the output
+   * is the value of the component of the displacement.
+   *
+   * @note This method must be called prior to completeSetup()
```

Comment:
Can/should this be checked at runtime?

Reply:
Yeah, that's a good idea.  @btalamini is doing a more complete overhaul of this now.  Could this check be added to your Dirichlet BC overhaul, Brandon?

Reply:
We have these comments all over the place.  At some point we can go through and have a check that people are not calling this after completeSetup(), but I think its out of the scope for this.